src/main/java/pages/BrowsePage.java

```diff
@@ -0,0 +1,65 @@
+package pages;
+
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.iOSXCUITFindBy;
+import utils.SwipeHelper;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BrowsePage extends BasePage {
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "search")
+    private MobileElement searchInput;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(xpath = "//android.widget.ScrollView[@content-desc=\"browseSectionsList\"]/android.view.ViewGroup/android.view.ViewGroup/android.view.ViewGroup/android.widget.TextView")
```

Comment:
What are you searching here? because if it is all the browse section you had the accesibility id to match direcly

Reply:
I am looking for the different topics of the movies, but Appium does not show value in accessibility_id

<img width="747" alt="Topicos" src="https://user-images.githubusercontent.com/75274560/108865805-9ababf80-75c1-11eb-8712-993ae1559e5c.png">